packages/twitch-eventsub/src/Events/EventSubChannelRewardRemoveEvent.ts

```diff
@@ -0,0 +1,238 @@
+import { Enumerable } from '@d-fischer/shared-utils';
+import type { ApiClient, HelixUser } from 'twitch';
+
+export interface EventSubChannelRewardRemoveEventData {
+	id: string;
+	broadcaster_user_id: string;
+	broadcaster_user_name: string;
+	is_enabled: boolean;
+	is_paused: boolean;
+	is_in_stock: boolean;
+	title: string;
+	cost: number;
+	prompt: string;
+	is_user_input_required: boolean;
+	should_redemptions_skip_request_queue: boolean;
+	cooldown_expires_at: string | null;
+	redemptions_redeemed_current_stream: number | null;
+	max_per_stream: EventSubMaxStreamData;
+	max_per_user_per_stream: EventSubMaxStreamData;
+	global_cooldown: EventSubCooldownData; // eslint-disable-line @typescript-eslint/naming-convention
+	background_color: string;
+	image: EventSubRewardImageData;
```

Comment:
The fact that the values can be null in `image` but not in `default_image` can be modeled with some TS helper type IIRC, I think it would be something like `default_image: Required<EventSubRewardImageData>`.